test/index.js

```diff
@@ -25,6 +25,13 @@ const expect = Code.expect;
 
 describe('Bananas', () => {
 
+    const originalPost = Wreck.post;
+    lab.afterEach((done) => {
```

Comment:
I just never use it. I like my tests to be completely atomic without leaking state outside the `it()` section.